test/query/query.test.ts

```diff
@@ -65,6 +65,29 @@ describe('query/query', () => {
       assert.equal((<SpecQueryModel>result.items[0]).specQuery.mark, 'tick');
       assert.equal((<SpecQueryModel>result.items[1]).specQuery.mark, 'point');
     });
+
+    it('should return a plot with name on x', () => {
```

Comment:
So many problems with this line!
1) The test description is too low level, doesn't reflect the test goal. 
2) Why do you put it under "nested query" describe, which is not relevant to this test at all.